src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Application {
+    private static final int arrayLenght = 6;
+    private static FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
Avoid using static non-final fields. Instead, create an instance of 'FigureSupplier' in the 'main' method to avoid potential issues with shared state.